applications/RomApplication/python_scripts/rom_manager.py

```diff
@@ -0,0 +1,526 @@
+import KratosMultiphysics
+from KratosMultiphysics.RomApplication.rom_testing_utilities import SetUpSimulationInstance
+from KratosMultiphysics.RomApplication.calculate_rom_basis_output_process import CalculateRomBasisOutputProcess
+from KratosMultiphysics.RomApplication.randomized_singular_value_decomposition import RandomizedSingularValueDecomposition
+import numpy as np
+import importlib
+import json
+
+
+
+
+class RomManager(object):
+
+    def __init__(self,project_parameters_name, general_rom_manager_parameters, CustomizeSimulation, UpdateProjectParameters):
+        #FIXME:
+        # - Use a method (upcoming) for smothly retrieving solutions. In here we are using the RomBasisOutput process in order to store the solutions
+        # - There is some redundancy between the methods that launch the simulations. Can we create a single method?
+        # - We must use the Hyper-Reduced model part for the HROM simulations. (The one in which we have eliminated the non selected elements and conditions)
+        # - Not yet paralellised with COMPSs
+        self.project_parameters_name = project_parameters_name
+        self.general_rom_manager_parameters = general_rom_manager_parameters
+        self.rom_training_parameters = self._SetRomTrainintParameters()
+        self.hrom_training_parameters = self.SetHromTrainingParameters()
+        self.CustomizeSimulation = CustomizeSimulation
+        self.UpdateProjectParameters = UpdateProjectParameters
+
+
+
+    def Fit(self, mu_train=None):
+        if mu_train is None:
+            mu_train = ['single case with parameters already contained in the ProjectParameters.json and CustomSimulation']
+        #######################
+        ######  Galerkin ######
+        if self.general_rom_manager_parameters["projection_strategy"].GetString() == "galerkin":
+            training_stages = self.general_rom_manager_parameters["rom_stages_to_train"].GetStringArray()
+            if any(item == "ROM" for item in training_stages):
+                fom_snapshots = self.LaunchTrainROM(mu_train)
+                self._ChangeRomFlags(simulation_to_run = "GalerkinROM")
```

Comment:
My suggestion for organizing these flags would be the following:
Galerkin:

- run_galerkin_rom
- train_galerkin_hrom
- run_galerkin_hrom

LSPG:

- run_lspg_rom

Petrov-Galerkin:

train_petrov_galerkin_rom

- run_petrov_galerkin_rom
- train_petrov_galerkin_hrom
- run_petrov_galerkin_hrom

However, since this restructuring won't be visible to the user, it could be implemented in a future pull request.

```suggestion
                self._ChangeRomFlags(simulation_to_run = "GalerkinROM") #TODO: Rename flags.
```